Composer/packages/client/src/utils/pageLinks.ts

```diff
@@ -67,6 +71,18 @@ export const topLinks = (projectId: string, openedDialogId: string) => {
     links = links.filter((link) => link.to !== '/home');
   }
 
+  if (pluginPages.length > 0) {
+    pluginPages.forEach((p) => {
+      links.push({
+        to: `page/${p.id}`,
+        iconName: p.icon ?? 'StatusCircleQuestionMark',
+        labelName: p.label,
+        exact: true,
+        disabled: false,
```

Comment:
could possibly add some logic to the extension config here that says:

```
{
  "views": {
    "page": {
      "label": "Sample UI Plugin",
      ...
      "enabled": <some-condition>,
    }
  }
}
```

where `<some-condition>` could be something like `"botProjectOpen"` or another special Composer state that extensions can key off of.